.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-package-lock.json

```

Comment:
Sorry, I think we should ignore the package-lock, I'd rather master failed and know what our users see than requiring an update to snapshots than a false sense of security.

https://www.twilio.com/blog/lockfiles-nodejs

Reply:
That's fair, one thing I'll say is that with the lockfile and your current Dependabot setup, the only difference would be that builds would be deterministic until you vet the Dependabot update (which are low-maintenance given the test coverage you have), so it would be easier to flag changes like the one in `axe-core` that caused this (instead of going through their package's build history to match dates, which may not always be possible).

Anyhow, I'll defer to your judgment on that. :+1:  Reverting the lockfile bit altogether and leaving only the snapshot fix.

Reply:
Done and done :rocket: